tap_mailchimp/schemas/list_segment_members.json

```diff
@@ -110,6 +110,8 @@
         "string"
       ]
     },
+    "interests": {
```

Comment:
Is the intent here to let this field through and we are unsure of the type?

Reply:
@luandy64 Yes, it's some sort of object according to the docs, but we don't know what properties it has and we don't have any client data with it coming across.